ui/app/pages/settings/settings.component.js

```diff
@@ -136,7 +136,9 @@ class SettingsPage extends PureComponent {
           className={c({ 'settings-page__subheader--link': initialBreadCrumbRoute })}
           onClick={() => initialBreadCrumbRoute && history.push(initialBreadCrumbRoute)}
         >{subheaderText}</div>
-        {breadCrumbTextKey && <div><span>{'> '}</span>{t(breadCrumbTextKey)}</div>}
+        {breadCrumbTextKey && <div
+          className={c({ 'settings-page__subheader--break': initialBreadCrumbRoute })}
```

Comment:
Should this be contingent upon `initialBreadCrumbRoute`? It looks like you're following the example set a couple lines up, which unfortunately isn't a very good example.

`classnames` expects arguments that are either strings, or objects with Boolean values. The example on line `136` really shouldn't be passing a string as the value - it should be using `isMyAccountsPage` instead (which is a Boolean, and which is `true` in every case that `initialBreadCrumbRoute` is truthy).

In this case, I think we want this style to always be applied? So you can just set it directly as the `className` property instead.

Reply:
updated based on feedback